internal/controller/helmrelease_controller.go

```diff
@@ -391,6 +392,13 @@ func (r *HelmReleaseReconciler) reconcileRelease(ctx context.Context, patchHelpe
 	obj.Status.LastAttemptedRevisionDigest = ociDigest
 	obj.Status.LastAttemptedConfigDigest = chartutil.DigestValues(digest.Canonical, values).String()
 	obj.Status.LastAttemptedValuesChecksum = ""
+	// Keep track of the post-renderers digest used during the last reconciliation.
+	// This is used to determine if the post-renderers have changed.
+	oldPostRenderersDigest := obj.Status.LastAttemptedPostRenderersDigest
+	// Update the post-renderers digest with the current post-renderers in the spec.
+	if obj.Spec.PostRenderers != nil {
+		obj.Status.LastAttemptedPostRenderersDigest = postrender.Digest(digest.Canonical, obj.Spec.PostRenderers).String()
+	}
```

Comment:
What happens if the previous `PostRenderers` were set but in the current version I remove them?

Reply:
As long as it is set, we want to reflect it. We need to keep the logic around the `object` itself in the `reconciler` code. I will push an update shortly.